<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>931</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>931</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>291</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>201</height>
      </rect>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Solidering </string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_bistate">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Bistate constant temp. controll</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_tristate">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Tristate constant temp controll</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_tempCheck">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature check</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_tempCurve">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>External temperature curve file</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>PID temperature controll</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QwtPlot" name="qwtPlot">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserLOG">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>641</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bStart">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close1">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>530</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QDoubleSpinBox" name="dSpinBox_D">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_D">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>29</width>
       <height>160</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dSpinBox_P">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_I">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>29</width>
       <height>160</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dSpinBox_I">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="verticalSlider_P">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>29</width>
       <height>160</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>I</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>290</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QDial" name="dial_PWM">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
    </widget>
    <widget class="QDial" name="dial_Temp">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PWM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Temp</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dSpinBox_PWM">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dSpinBox_Temp">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select type of work:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_loadFile">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>530</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load curve</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt/qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
